### PR TITLE
TINKERPOP-1019 Replaced all system.out.println and system.out.error with logger.

### PR DESCRIPTION
**Pull request by n-tran**
Removed the `muteTestLog` system property because logging can now be controlled by config file.

Ran `mvn clean install` and `mvn clean install -Dci`

Logging appear appropriate in both cases.

### CODE REVIEW

File: gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/GremlinProcessRunner.java [hunk 1]
```diff
@@ -54,7 +56,7 @@ public void runChild(final FrameworkMethod method, final RunNotifier notifier) {
             } catch (Throwable e) {
                 if (validateForGraphComputer(e)) {
                     eachNotifier.fireTestIgnored();
-                    System.err.println(e.getMessage());
+                    logger.error(e.getMessage());
                     ignored = true;
```
Comment (spmallette) [inline gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/GremlinProcessRunner.java]: @dkuppitz perhaps on merge we could change this to WARN level?  this situation doesn't really indicate a "failure" does it?

**Comment by spmallette** [comment]
lgtm - travis passes and logging line counts are down by about 2000 lines or so

vote: +1

File: gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/GremlinProcessRunner.java (hunk 1 above)
Comment (dkuppitz) [inline gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/GremlinProcessRunner.java]: Yeah, I think WARN would be fine too. INFO is what's causing trouble.

File: gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java [hunk 2]
```diff
@@ -71,7 +73,7 @@
  * @author Stephen Mallette (http://stephen.genoprime.com)
  */
 public class GremlinDriverIntegrateTest extends AbstractGremlinServerIntegrationTest {
-
+    private static final Logger logger = LoggerFactory.getLogger(AbstractGremlinServerIntegrationTest.class);
```
Comment (dkuppitz) [inline gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java]: This should be `LoggerFactory.getLogger(GremlinDriverIntegrateTest .class)`, right?

Comment (spmallette) [inline gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java]: well noted - i missed that one in my review.

**Comment by dkuppitz** [comment]
Looks good now; I guess the merge conflict is just a minor issue.

VOTE: +1

**Comment by spmallette** [comment]
i'll deal with the merge - probably not a big deal.

**Comment by twilmes** [comment]
VOTE: +1

**Comment by spmallette** [comment]
merged to master - thanks @n-tran

**Pull request merged**